wiki/Options/en.md

```diff
@@ -529,609 +166,141 @@ Note: Your current playing song on your jukebox will be used.
-<td><p><a href="Media:OP_J_Line.jpg" title="wikilink">Make Judgement Line slightly &quot;brighter&quot;.</a></p></td>
-</tr>
-<tr class="odd">
-</tr>
-</tbody>
-</table>
+| Name                                  | Effect                                                                                          |
+|---------------------------------------|-------------------------------------------------------------------------------------------------|
+| Key Count                             | Normal settings for 1K ~ 9K, 5K ~ 9K Co-op                                                      |
+| Co-op Layout                          | Settings after applying co-op mod (currently apply to 1K ~ 9K. 10K ~ 18K is still in progress). |
+| Column Style (6K and 8K only)         | Adjust column style?                                                                            |
+| Bind alternate special keys           | (When using left/right column style) Bind the alternate special key.                            |
+| Vertically flip playfield (DDR style) | Move the judgement area and keynotes above the screen instead of below.                         |
+| Show Judgement line                   | Make judgement line slightly "brighter".                                                        |
 
 ### Other
 
-<table style="width:36%;">
-<colgroup>
-<col width="12%" />
-<col width="24%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th><p>Name</p></th>
-<th><p>Effect</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>OS TabletPC support.</p></td>
-<td><p>Enable if you use a graphic tablet or tablet PC and your clicks aren't handled correctly.</p></td>
-</tr>
-<tr class="even">
-<td><p>Joystick support</p></td>
-<td><p><strong>[<a href="osu!mania" class="uri" title="wikilink">osu!mania</a> only]</strong><br />
-Enable this if you use a controller or joystick to play osu!mania.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Wiimote/TaTaCon Drum Support</p></td>
-<td><p>Enable support for Nintendo's Wii Taiko Drum controller and Wiimote. Pair your device first (via Bluetooth) before enabling it.</p></td>
-</tr>
-</tbody>
-</table>
-
-![](OP Editor thumb.jpg "fig:OP Editor thumb.jpg") Editor section
------------------------------------------------------------------
-
-<img src="Options Editor.jpg" title="Editor section" alt="Editor section" width="300" />
+| Name                         | Effect                                                                                                           |
+|------------------------------|------------------------------------------------------------------------------------------------------------------|
+| OS TabletPC support          | Enable if your use a graphic tablet or tablet PC and your clicks aren't being handled correctly.                 |
+| Joystick support             | (osu!mania only) Enable this if you use a joystick or controller to play osu!mania.                              |
```

Comment:
I've looked it up and it is apparently used for osu!mania as well. How? I have no fricking clue.